src/layouting/boundingbox.jl

```diff
@@ -172,14 +172,14 @@ function boundingbox(
                 inkboundingbox(FreeTypeAbstraction.get_extent(font, c))
             end
 
-            scaled_bb = bb_unitspace * scale
+            scaled_bb = rectmult(bb_unitspace, scale)
```

Comment:
I moved this to GeometryBasics where it should live! You need the newest version for it though

Reply:
So would be nice to revert this ;)